jobs/build/ocp/Jenkinsfile

```diff
@@ -642,11 +661,81 @@ Please direct any questsions to the Continuous Delivery team (#aos-cd-team on IR
                 sh "ose_images.sh --user ocp-build build_container --branch rhaos-${BUILD_VERSION}-rhel-7 --group base --repo https://raw.githubusercontent.com/openshift/aos-cd-jobs/master/build-scripts/repo-conf/aos-unsigned-building.repo"
             }
 
-            buildlib.oit """
+            waitUntil {
+              try {
+                exclude = ""
+                if ( BUILD_EXCLUSIONS != "" ) {
+                  exclude = "-x ${BUILD_EXCLUSIONS} --ignore-missing-base"
+                }
+                buildlib.oit """
 --working-dir ${OIT_WORKING} --group openshift-${BUILD_VERSION}
+${exclude}
 images:build
 --push-to-defaults --repo-type unsigned
 """
+                return true // finish waitUntil
+              }
+              catch ( err ) {
+                record_log = buildlib.parse_record_log( OIT_WORKING )
+                builds = record_log['build']
+                failed_map = [:]
+                for(i = 0; i < builds.size(); i++) {
+                  distgit = build[i]['distgit']
+                  if ( builds[i]['status'] == '-1' ){
+                    task_url = "No Brew task available, see jenkins log."
+                    if ( build[i].containsKey('task_url') ) {
+                      task_url = build[i]['task_url']
+                    }
+                    failed_map[distgit] = task_url
+                  }
+                  else {
+                    // build may have succeeded later. If so, remove.
+                    if ( failed_map.containsKey(distgit) ) {
```

Comment:
No need to check. Groovy doesn't throw an exception if it is not present.

Reply:
Good to know.